cleanup/style guide: update descriptions section (#12313)
tldr-pages/tldr: contributing-guides/style-guide.md

 `ls`
```

### Wording

- All descriptions **must be concise**.
- Avoid using the page title in the description (e.g. use `A sketching and painting program designed for digital artists` instead of `Krita is a sketching and painting program designed for digital artists`) unless the program name differs from the executable name (e.g. `rg` and Ripgrep).
- Avoid mentioning that the program is used on the command-line (e.g. use `Ripgrep is a recursive line-oriented search tool` instead of `Ripgrep is a recursive line-oriented CLI search tool`).

> - For example, when writing documentation for `cd`, a tool to check out and work on a specific directory in the Terminal or Command Prompt, **do not** write a lengthy description such as:

Reviewer: Perhaps this will read better?
```suggestion
- For example, when writing documentation for `cd`, a tool to change the current working directory in the Terminal or Command Prompt, **do not** write a lengthy description such as:
```

```md

> > `cd` is a system tool, available in Windows, macOS, and Linux, to check out a specific directory to get things done in the Command Prompt, Terminal, and PowerShell.

Reviewer: I know this is a counter-example, but accuracy still bugs me haha 
```suggestion
> `cd` is a system tool, available in Windows, macOS, and Linux, to change the current working directory to get things done in the Command Prompt, Terminal, and PowerShell.
```
In a sense  this example is also a counter-example ref objectivity too.

```

It should instead be simplified to make it easier for everyone to read:

```md
> Change the current working directory.
```

### Formatting

> - Brand and project names can be capitalized in the description whenever applicable (e.g. use `A tool for interacting with a Git repository.` instead of ``A tool for interacting with a `git` repository.``).

Reviewer 2: ```suggestion
- Proper names should be capitalized in the description whenever applicable (e.g. use `A tool for interacting with a Git repository.` instead of ``A tool for interacting with a `git` repository.``).
```

This would then also include things that are neither brand- nor project names, e.g. Netpbm.

- Acronym expansions (i.e. protocols, tools, etc) must not be translated unless there is a recognized native equivalent for them.

> - When documenting keycaps or a keyboard shortcut for a utility it is suggested to wrap them in backticks to make them stand out in the description (i.e. ``Print the last lines of a given file and keep reading it until `Ctrl + C`:``). Alternatively, you can document them as a separate command and optionally highlight them as placeholders (i.e. `:wq{{Enter}}` or `:wq<Enter>` or `:wq(Enter)`).

Reviewer 2: Should we work towards unifying the notation for key strokes?

Reviewer 3: Probably yeah, the current methods are very inconsistent.

Author: I'm favor of standardizing to:
- `Backticks` for non-translatable keycaps/keyboard shortcuts
- \<Angle brackets\> for translatable keycaps/keyboard shortcuts

Reviewer: @vitorhcl this seems sensible to me. Go for it!

Author: Currently, the client specification says nothing about translatable keycaps or keyboard shortcuts. On second thought, angle brackets are already used in CommonMark for autolinks and raw HTML, so maybe it would be better for the clients if we used a custom syntax like we do with the placeholders, such as "[[ "and "]]", or "<<" and ">>" or another syntax?

Author: @kbdharun @sebastiaanspeck @sbrl any opinion on my suggestion above?
